.rubocop.yml

```diff
@@ -1,32 +1,39 @@
 Rails/CreateTableWithTimestamps:
   Enabled: false
 Rails/HasManyOrHasOneDependent:
-  Enabled: false
+  Enabled: true
+# if a codebase is on an older version on rails override this in the local rubocop.yml
 Rails/HttpPositionalArguments:
-  Enabled: false
+  Enabled: true
 Rails/LexicallyScopedActionFilter:
   Enabled: false
 Rails/Presence:
-  Enabled: false
+  Enabled: true
 Rails/FilePath:
   EnforcedStyle: arguments
-# We should enable this later but there is currently a bug in rubocop which
-# breaks this
-# Rails/RequestReferer:
-#   EnforcedStyle: referrer
-#
-# Also only allow update_all because it's fast.
-# Rails/SkipsModelValidations:
-#   Blacklist:
-#     - decrement!
-#     - decrement_counter
-#     - increment_counter
-#     - toggle!
-#     - touch
-#     - update_attribute
-#     - update_column
-#     - update_columns
-#     - update_counters
-
+Rails/RequestReferer:
+  EnforcedStyle: referrer
+# these paths are self explanatory in a Rails context and do not class comments
+Style/Documentation:
+  Exclude:
+    - 'db/migrate/*'
+    - 'test/**/*'
+    - 'config/application.rb'
+    - 'app/policies/*_policy.rb'
+    - 'app/helpers/*.rb'
```

Comment:
@tye-jjohnson, for `'app/helpers/*.rb'`, we didn't plan to enforce the documentation because it is a module? 
Somehow feel it will be helpful to document those view helpers if we have customized ones that might not correspond to controllers.

Reply:
@jessie19 I think this is for documenting the object, not the methods within it, right? In which case I would agree that the class itself doesn't need documentation, but its methods do.

Reply:
I agree it is helpful to document helper methods, this disables documentation of the Helpers module itself which doesn't have any greater meaning than a helper module for your views.

Reply:
Thanks for the clarification, resolve this thread!